Sunrise/Models/ProductProjection.swift

```diff
@@ -0,0 +1,37 @@
+//
+// Copyright (c) 2016 Commercetools. All rights reserved.
+//
+
+import ObjectMapper
+
+struct ProductProjection: Mappable {
+
+    // MARK: - Properties
+
+    var id: String?
+    var name: [String: String]?
```

Comment:
As a suggestion: in this context, `typealias LocalizedString = [String: String]` could improve readability.